internal/utils/testutil/binary.go

```diff
@@ -24,15 +24,27 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-func GinkgoGetHorusecCmd(horusecCmd string) *exec.Cmd {
+type HorusecCmd string
```

Comment:
Changes to this file must be discarded before merging.

Reply:
Adjusted!